generate.ts

```diff
@@ -309,13 +455,15 @@ const getRepositories = async (
     fs.writeFileSync("./generated.json", JSON.stringify(data));
     console.log("Generated generated.json");
 
-    // Update firstissue.json with new list of repositories
-    firstissue.repositories = data.repositories
-      .map((repo) => `${repo.owner}/${repo.name}`)
-      // Sort alphabetically
-      .sort((a, b) => a.localeCompare(b));
-    fs.writeFileSync("./firstissue.json", JSON.stringify(firstissue, null, 2));
-    console.log("Generated firstissue.json");
+    // Disabled: why should we require the file? Only to ensure alphabetical order?
```

Comment:
That, and repos that don't meet criteria anymore are removed and the file is overwritten.

Reply:
@lucavallin would it be alright to remove this logic?
Now that we have more sources/providers it's kinda harder to recreate the file structure, and at this point we wouldn't even have all the info needed to recreate it (e.g. we don't know where a project is coming from).
Moreover, I think it would be sufficient to "hide" a project (so don't show any issue from it) when it doesn't meet the criteria anymore. I think it'd be kinda "bad" to have to re-add the project multiple times (thinking about new projects that may not be super "active" or "open").

WDYT?

Reply:
@lucavallin that sounds totally good to me, I'll do that 🙂

Reply:
@lucavallin this turned out to be quite "intricate" since we have multiple reasons why a project is not added (namely stars count and  issues count) and we have different places where we do filter those out (for GitHub is a mix of GraphQL query and manual filter, for GitLab is only manual filter).

Probably, a good way to handle this would do all the queries and then iterate over the repos coming from a source and see if it's there or not, and log those results. WDYT? Anyway, in the spirit of speeding this up I would be tempted to not block this PR only for that, we can fix it as an iteration, but up to you to decide 🙂

Reply:
Yep, sounds good, feel free to do that as a follow-up (please create an issue for that so we don't forget).

Reply:
@lucavallin, follow-up issue created 👍